libraries/pbrlib/genglsl/lib/mx_microfacet_diffuse.glsl

```diff
@@ -1,7 +1,7 @@
 #include "mx_microfacet.glsl"
 
-// Based on the implementation of Oren-Nayar diffuse in Open Shading Language.
-// https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/blob/main/src/testrender/shading.cpp
+// Qualitative Oren-Nayar diffuse with improvements from Fujii:
+// https://mimosa-pudica.net/improved-oren-nayar.html
 float mx_oren_nayar_diffuse(float NdotV, float NdotL, float LdotV, float roughness)
 {
     float s = LdotV - NdotL * NdotV;
```

Comment:
To match the Fujii model, the code for `s` needs to be:
```glsl
float stinv = (s > 0.0f) ? s / max(NdotL, NdotV) : s;
//float stinv = (s > 0.0f) ? s / max(NdotL, NdotV) : 0.0;   // <-- not this
```

Reply:
@portsmouth I was aiming to match @fpsunflower's implementation of QON in Open Shading Language, which has the `0.0` fallback as well.  Should the change that you propose be made there as well?